_config.yml

```diff
@@ -26,7 +26,7 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: 18F Delivery: Partnership Playbook
+- text: 18F Delivery":" Partnership Playbook
```

Comment:
Err, this is `_config.yml`... I meant the `title:` property of the index page also needs the quotes.